core/src/main/java/io/grpc/internal/GrpcUtil.java

```diff
@@ -179,6 +179,20 @@
   public static final long DEFAULT_KEEPALIVE_TIMEOUT_NANOS = TimeUnit.MINUTES.toNanos(2);
 
   /**
+   * The default delay in nanos for server keepalive.
+   *
+   * @since 1.3.0
+   */
+  public static final long DEFAULT_SERVER_KEEPALIVE_TIME_NANOS = TimeUnit.HOURS.toNanos(2L);
+
+  /**
+   * The default timeout in nanos for a server keepalive ping request.
+   *
+   * @since 1.3.0
+   */
+  public static final long DEFAULT_SERVER_KEEPALIVE_TIMEOUT_NANOS = TimeUnit.SECONDS.toNanos(20L);
```

Comment:
Shouldn't the "unset" variable be here too?  

public static final long SERVER_KEEPALIVE_TIME_NANOS_DISABLED = Long.MAX_VALUE;